esx_service/cli/vmdkops_admin.xml

```diff
@@ -0,0 +1,571 @@
+                            <string/>
+                        </field>
+                    </structure>
+                </list>
+            </output-spec>
+            <has-updates value="true"/>
+            <format-parameters>
+                <formatter>table</formatter>
+                <format-parameter name="fields:vdvs">Uuid, Name</format-parameter>
+            </format-parameters>
+            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml vmgroup vm ls --name=$val{name}</execute>
+        </command>
+        <!-- vmgroup access commands -->
+        <command path="vdvs.vmgroup.access.add">
+            <description>Add a datastore access for a vmgroup</description>
+            <input-spec>
+                <parameter name="name" type="string" required="true">
+                    <description>The name of the vmgroup</description>
+                </parameter>
+                <parameter name="datastore" type="string" required="true">
+                    <description>Datastore name</description>
+                </parameter>
+                <parameter name="allow-create" type="bool" required="false">
+                    <description>Allow create and delete on datastore if set</description>
+                </parameter>
+                <parameter name="volume-maxsize" type="string" required="false">
+                    <description>Maximum size of the volume that can be created</description>
+                </parameter>
+                <parameter name="volume-totalsize" type="string" required="false">
+                    <description>Maximum total size of all volume that can be created on the datastore for this vmgroup</description>
+                </parameter>
+            </input-spec>
+            <output-spec>
+                <string/>
+            </output-spec>
+            <format-parameters>
+                <formatter>simple</formatter>
+            </format-parameters>
+            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml vmgroup access add --name=$val{name} --datastore=$val{datastore} $if{allow-create, --allow-create} $if{volume-maxsize, --volume-maxsize=$val{volume-maxsize}} $if{volume-totalsize, --volume-totalsize=$val{volume-totalsize}} </execute>
+        </command>
+        <command path="vdvs.vmgroup.access.set">
+            <description>Modify datastore access for a vmgroup</description>
+            <input-spec>
+                <parameter name="name" type="string" required="true">
+                    <description>The name of the vmgroup</description>
+                </parameter>
+                <parameter name="datastore" type="string" required="true">
+                    <description>Datastore name</description>
+                </parameter>
+                <parameter name="allow-create" type="bool" required="false">
+                    <description>Allow create and delete on datastore if set</description>
```

Comment:
"Allow create and delete on datastore if set to true". This value can also be set to "false" by user.